test_utils/factories.py

```diff
@@ -393,6 +395,7 @@ class Meta:
     transaction_id = factory.LazyAttribute(lambda x: FAKER.uuid4())
     enterprise_course_enrollment = factory.SubFactory(EnterpriseCourseEnrollmentFactory)
     is_revoked = False
+    fulfillment_type = FulfillmentTypes.LEARNER_CREDIT
```

Comment:
Another set of unrelated changes here.  I just noticed that all the `LearnerCreditEnterpriseCourseEnrollment` objects in unit tests had the wrong fulfillment_type.